1 Holiday Rental Property Management Tool/submissions/enricop89/README.md

```diff
@@ -0,0 +1,21 @@
+# Holiday Rental Property Management Tool
+
+For this project I imagined to host the client side application in an S3 bucket backed by a Cloudfront distribution to enforce HTTPS, global distribution and reduced latency. The client side application will have two main users: guests and hosts. Hosts can login using AWS Cognito, Guests can only see the property and book a room.
+
+The Host flow is the following:
+
+- Host signs up for the platform and chooses his own subdomain
+- if available, a new record is created on S3 and a dedicated user pool in AWS Cognito
```

Comment:
Would the subdomain be created by s3 or would you user Route53?

Reply:
Route53 that points to the cloudfront distribution